core/src/components/BlockView.tsx

```diff
@@ -15,11 +14,11 @@ interface BlockViewProps<T extends string = LanguageCode> {
   timeStyle?: TimeStyle;
   textStyle?: TextStyle;
   FocusButton?: ({
-    isFocus,
-    setIsFocus,
+    isFocused,
+    setIsFocused,
   }: {
-    isFocus: boolean;
-    setIsFocus: React.Dispatch<React.SetStateAction<boolean>>;
+    isFocused: boolean;
+    setIsFocused: React.Dispatch<React.SetStateAction<boolean>>;
   }) => JSX.Element;
```

Comment:
Instead of passing the focus button itself, it would be better to pass` isFocus` and `setIsFocused` as props. I'll try later.